nats-api/src/main/kotlin/com/example/ajaxonboardingproject/NatsSubject.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.ajaxonboardingproject
+
+object NatsSubject {
+    const val ADD_NEW_CINEMA_HALL_SUBJECT = "v2.ajax_onboarding_project.input.reqreply.cinema_hall.add"
```

Comment:
nice to see well-known style :)

Reply:
@ajax-shulepa-a 

But we don't have multiple nested classes here))